src/scene.ts

```diff
@@ -17,7 +17,8 @@ import { Model } from './model';
 import { Splat } from './splat';
 import { Camera } from './camera';
 import { CustomShadow as Shadow } from './custom-shadow';
-import { Grid } from './grid';
+// import { Grid } from './grid';
```

Comment:
Delete?

Reply:
And can `grid.ts` be deleted too?

Reply:
We can delete these. I kept them incase users would like the option of old grid or incase we need to revert this one. I'll remove the code once we're happy to keep the new grid.